notebooks/TFoS.ipynb

```diff
@@ -18,18 +18,24 @@
     "import shlex\n",
     "from imp import reload\n",
     "from tensorflowonspark import TFCluster\n",
-    "from pyspark import SparkContext"
+    "from pyspark import SparkContext\n",
+    "from pyspark.conf import SparkConf"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
-   "outputs": [],
+   "cell_type": "raw",
+   "metadata": {},
    "source": [
-    "sc = SparkContext.getOrCreate()\n",
+    "# GPU Comfig\n",
```

Comment:
Minor Nit: GPU **Config** :)

Would you please move out the GPU-specific code into its own cell so we can just one-shot `[Ctrl|Cmd]+#` uncomment it when we'd  like to run on with GPUs?

Reply:
Also, if you should so feel up to the challenger, we should adapt it to v1.3.2 so we don't have to perform an additional `git checkout v1.3.0` of https://github.com/yahoo/TensorFlowOnSpark before running the the Notebook.

Brownie Points: Adapt the Notebook to a slide show with the [RISE](https://github.com/damianavila/RISE) extension we already have bundled :)